src/components/weather/currenetWeather.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import moment from "moment";
+
+import { toCelcius, toFahrenheit } from "utils/tempConverter";
+
+function CurrentWeather({ current, tempPresenter }) {
+  useEffect(() => {}, []);
```

Comment:
Does this line have an effect on the component? No worries if it's leftover from a previous effect, just curious!